File: components_page/components/input/number_input.py
import dash_bootstrap_components as dbc
from dash import html

number_input = html.Div(
    [
        html.P("Type a number outside the range 0-10"),
        dbc.Input(type="number", min=0, max=10, step=1),
    ],
    id="styled-numeric-input",
)

File: components_page/components/input/number_input.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

number_input <- htmlDiv(
  list(
    htmlP("Type a number outside the range 0-10"),
    dbcInput(type = "number", min = 0, max = 10, step = 1)
  ),
  id = "styled-numeric-input",
)
